src/containers/Map/index.tsx

```diff
@@ -66,16 +68,17 @@ const Map: React.FC<Props> = (props) => {
 
         {/* Marker */}
         <Marker
-          position={DEFAULT_CENTER}
+          position={position || DEFAULT_CENTER}
           icon={DEFAULT_LOGO}
-          onClick={() => onClickMarker(DEFAULT_CENTER)}
+          onClick={() => onClickMarker(position)}
         />
 
         {/* Map Info */}
-        {position && (
+        {isShow && (
```

Comment:
```suggestion
        {isShow && (
```
why not to replace `isShow` to `isOpenMapInfo`